### PR TITLE
✨ Support signup and login via emailed magic link

### PR DESCRIPTION
**Pull request by victorlora**
I think we need to set `NEXT_PUBLIC_API_ROOT` to the right IP Address. Not sure where that is managed for Vercel - assuming... Vercel.

image

### CODE REVIEW

File: pages/login.tsx [hunk 1]
```diff
@@ -43,16 +49,30 @@ const Login: NextPage = () => {
...
+      {!credentialMode && (
+        <>
+          <input
+            {...register('email')}
+            className="bg-gray-700 border"
+            type="email"
+          />
+        </>
+      )}
+      {credentialMode && (
```
Comment (ethanmick) [inline pages/login.tsx:61]: Do we even need credential mode? I'll be quite happy to just throw this out and only do magic links.

**Review by ethanmick — APPROVED** [review]
I'll add the ENV Var to the prod env :)

File: pages/login.tsx (hunk 1 above)
Comment (victorlora) [reply pages/login.tsx:61]: I’m down to just throw it out. We can always bring it back later if needed. I’ll just need your email address to add to MG until we have a valid domain. MG only allows sending to verified emails from their sandbox environment.

**Comment by ethanmick** [comment]
It's been added, this can be merged.

**Pull request merged**